src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.html

```diff
@@ -56,7 +56,8 @@ <h3 class="font-bolt order-title">{{ 'order-details.order-title' | translate }}<
               <p class="total-content" *ngIf="pointsUsed">
                 <span>{{ 'order-details.bonuses' | translate }} </span
                 ><span>
-                  <strong class="money-saved">-{{ pointsUsed }} {{ 'order-details.currency' | translate }}</strong></span>
+                  <strong class="money-saved">-{{ pointsUsed }} {{ 'order-details.currency' | translate }}</strong></span
+                >
```

Comment:
move > to line 59

Reply:
also use currency pipe

Reply:
![image](https://user-images.githubusercontent.com/43093994/124005630-fbc82900-d9e1-11eb-8e6e-ea25d4810df0.png)